SpatialGDK/Source/SpatialGDKEditor/Public/SpatialTestSettings.h

```diff
@@ -46,6 +46,15 @@ class SPATIALGDKEDITOR_API FSpatialTestSettings
 	// Generated map override config base filename for generated maps applied to specific map, if exists
 	static const FString GeneratedOverrideSettingsBaseFilename;
 
+	/*
+	 * Creates the configuration file pathname for the given MapName, and returns it as an out parameter.
+	 * Returns true if the newly-created configuration file pathname exists on disk, false otherwise.
+	 */
+	static bool GenerateMapConfigurationFilename(const FString& MapName, FString& GeneratedConfigurationFilename);
```

Comment:
Not  needed if you follow my other suggestion but I prefer explicit prefixing out parameters with Out. E.g. `GeneratedConfigurationFilename` -> `OutGeneratedConfigurationFilename`

Reply:
Will keep in mind the Out prefixing, but in this case I will use your previous suggestion so it would not be needed.